### DISCUSSION TITLE
It won't let me change the color depth

### DISCUSSION THREAD

**Discussion by Jabnin** [Codectory/AutoActions#109] [Q&A]
The checkbox and drop-down remain grey.

**Comment by Codectory** [comment]
Only Nvidia cards are supported right now. Do you have a Nvidia graphic
card?

Tallcat ***@***.***> schrieb am Mi. 16. Feb. 2022 um 02:43:

**Reply by Jabnin** [reply]
I have a 3080ti. It lets me change the HDR settings, resolution and refresh rate, but not the color depth.
